Instrument accessibility note (#1632)
act-rules/act-rules.github.io: _rules/audio-or-video-avoids-automatically-playing-audio-80f0bf.md

## Background

> The [instruments][instrument] used to pass this rule (if any), must meet all level A Success Criteria in order to fully satisfy [Success Criterion 1.4.2 Audio Control][sc142]. These extra requirements are left out of this rule, and of the definition of [instrument][], for the sake of simplicity.

Reviewer: ```suggestion
The [instruments][instrument] used to pass this rule (if any), must meet all level A Success Criteria in order to fully satisfy [Success Criterion 1.4.2 Audio Control][sc142]. These extra requirements are left out of this rule, and should be tested separately.
```
